### PR TITLE
Adding a link to backup config page, added placeholder to MinIO config

### LINKED ISSUE #4245: Note that the cert in backup helm chart needs to be base64 encoded
Someone on the Rancher Users Slack had trouble following this guide: https://rancher.com/docs/rancher/v2.6/en/backups/examples/#backup

They eventually realized it was because their cert was not base64 encoded in the backup yaml Helm chart. That should be noted in the docs.

### PR DESCRIPTION
**Pull request by jtravee**
Closes #4245

### CODE REVIEW

File: content/rancher/v2.6/en/backups/examples/_index.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ Encrypted backups can only be restored if the Restore custom resource uses the s
 
 This section contains example Backup custom resources.
 
+>**Note:** Refer to the [backup config reference page]({{<baseurl>}}/rancher/v2.6/en/backups/configuration/backup-config/) for more information on configuring the options below.
```
Comment (btat) [inline content/rancher/v2.6/en/backups/examples/_index.md:34]: This same update (with the appropriate text and link) needs to be applied to the corresponding area in the restore section as well.

File: content/rancher/v2.6/en/backups/examples/_index.md [hunk 2]
```diff
@@ -84,7 +86,7 @@ spec:
       credentialSecretNamespace: default
       bucketName: rancherbackups
       endpoint: minio.xip.io
-      endpointCA: LS0tLS1CRUdJTi3VUFNQkl5UUT.....pbEpWaVzNkRS0tLS0t
+      endpointCA: <base64-encoded-cert>
```
Comment (btat) [inline content/rancher/v2.6/en/backups/examples/_index.md:89]: This same update needs to be applied to the corresponding area in the restore section as well.

**Review by btat — CHANGES_REQUESTED** [review]

File: content/rancher/v2.6/en/backups/examples/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/backups/examples/_index.md:89]: Updated, ty!

File: content/rancher/v2.6/en/backups/examples/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.6/en/backups/examples/_index.md:34]: Updated, ty!

**Review by btat — APPROVED** [review]

**Pull request merged**